### PR TITLE
Fix formatting issue for diagnostic messages

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Changes to show border for diag messages and to fix padding issue. Previous change was setting background color at each <p> node that has messed the layout. I have updated this to group all diagnostic messages of same type and show it as row.

Reverted background color for error and updated warning color to lightyellow.

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
Snip from test run for  single line diagnostic.

image

**Comment by praveenkuttappan** [comment]
Snip of multi line diagnostics.
image

**Comment by praveenkuttappan** [comment]
Snip of multi line with multiple levels from a test run. Please ignore dummy lines.
image

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml [hunk 1]
```diff
@@ -23,33 +23,17 @@
 </tr>
 @if (Model.Diagnostics.Any())
 {
+    var errorDiags = Model.Diagnostics.Where(d => d.Level == 0 || d.Level == APIView.CodeDiagnosticLevel.Error);
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml:26]: Add a Default member to CodeDiagnosticLevel with value 0

**Comment by pakrym** [comment]
Can we make the link less bright/bold as well?

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml (hunk 1 above)
Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml]: Done

**Comment by praveenkuttappan** [comment]
I have removed bold font added for the details.

**Comment by pakrym** [comment]
Can you share a screenshot, please?

**Comment by praveenkuttappan** [comment]
Screenshot with updated font for details link.
image

**Comment by praveenkuttappan** [comment]
No change since last commit for multiple diag types. Screenshot from latest test run.
image

**Review by pakrym — APPROVED** [review]

**Pull request merged**